tools/json-to-module/main.js

```diff
@@ -124,14 +125,15 @@ async function assemble(args) {
 
     const contentPath = path.resolve(__dirname, "../../content");
 
+    const manifestOutcome = await assembleManifest(args);
     const outcomes = await Promise.allSettled([
         assembleScenes(args, contentPath),
         assembleTables(args, contentPath),
         // assembleActors(args),
         // assembleItems(args),
-        assembleManifest(args),
         assembleREADME(args),
     ]);
+    outcomes.push(manifestOutcome);
```

Comment:
Since the `assembleManifest` step will potentially prompt for user input (to confirm a URL), running it in parallel can mess with the stdout.